core_apps/institutes/forms.py

```diff
@@ -129,4 +129,125 @@ def save(self, institute):
         
         except Exception as e:
             return False, f"An error occurred: {e}"
-        
\ No newline at end of file
+        
+        
+ 
+class StudentForm(forms.Form):
+    first_name = forms.CharField(
+        max_length=200,
+        widget=forms.TextInput(attrs={'class': 'form-control', 'id': 'first_name'})
+    )
+    last_name = forms.CharField(
+        max_length=200,
+        widget=forms.TextInput(attrs={'class': 'form-control', 'id': 'last_name'})
+    )
+    email = forms.CharField(
+        max_length=200,
+        widget=forms.TextInput(attrs={'class': 'form-control', 'id': 'email'})
+    )
+    password = forms.CharField(
+        max_length=200,
+        widget=forms.TextInput(attrs={'class': 'form-control', 'id': 'password'})
+    )
+    department = forms.ModelChoiceField(
+        queryset=Department.objects.all(),
+        widget=forms.Select(attrs={'class': 'form-control', 'id': 'department'}),
+        empty_label="Select Department"
+    )
+    enrollment = forms.CharField(
+        max_length=200,
+        widget=forms.TextInput(attrs={'class': 'form-control', 'id': 'enrollment'})
+    )
+    def save(self, institute):
+        first_name = self.cleaned_data['first_name']
+        last_name = self.cleaned_data['last_name']
+        email = self.cleaned_data['email']
+        password = self.cleaned_data['password']
+        department = self.cleaned_data['department']
+        enrollment = self.cleaned_data['enrollment']
+
+        try:
+            with transaction.atomic():
+
+                # Create the User
+                user = AUTH_USER.objects.create(
+                    first_name=first_name,
+                    last_name=last_name,
+                    email=email,
+                    password=make_password(password),
+                )
+                # Create the Role
+                role = Role.objects.create(
+                    user=user,
+                    institute=institute,
+                    role_type=RoleType.STUDENT
+                )
+                
+                # Create the Student
+                Student.objects.create(
+                    role=role,
+                    department=department,
+                    enrollment_number=enrollment,
+                    institute=institute
+                )
+                
+            return True, "student created successfully"
+        
+        except Exception as e:
+            return False, f"An error occurred: {e}"
+               
+
+
+class AcademicSectionForm(forms.Form):
+    academicSection_name = forms.CharField(
```

Comment:
Use snake case for this `academicSection_name` (it should be like this academic_section_name)